internal/pkg/earpointer/analysis.go

```diff
@@ -382,7 +443,7 @@ func (vis *visitor) visitCall(call *ssa.CallCommon, instr ssa.Instruction) {
 		for _, calleeRet := range rets {
 			if len(calleeRet) == 1 { // non-tuple case
 				if mayShareObject(calleeRet[0]) {
-					vis.unifyLocals(callerRet, calleeRet[0])
+					contextSensitiveUnify(callerRet, calleeRet[0])
 				}
```

Comment:
nit: This has gotten a bit long. Add a continue here so the remainder can focus on len(calleeRet) != 1